src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerInitializationHostService.cs

```diff
@@ -115,15 +115,16 @@ public virtual async Task<string> Read(string uri)
             // But the blob pointed to by the uri will not exist until the container is specialized.
             // When the blob doesn't exist it just means the container is waiting for specialization.
             // Don't treat this as a failure.
-            var cloudBlockBlob = new CloudBlockBlob(new Uri(uri));
+            var blobClient = new BlobClient(new Uri(uri));
 
-            var blobRequestOptions = new BlobRequestOptions
+            if (await blobClient.ExistsAsync())
```

Comment:
You removed the retry specification here with no replacement? This was added to address issues we were seeing in prod.

Reply:
Do you have more details on these issues seen in prod? I'm not aware of an equivalent RetryPolicy for v12 storage. These methods removed the BlobRequestOptions. The closest thing I'm seeing is related to http headers for the BlobClient to add [BlobRequestConditions ](https://docs.microsoft.com/en-us/dotnet/api/azure.storage.blobs.models.blobrequestconditions?view=azure-dotnet)which don't have a similar RetryPolicy

Reply:
@balag0 made this addition originally and can comment more on whether it's still needed and what we should do